### ISSUE TITLE
Error trying the example

### ISSUE THREAD

**Issue by emiliomarin** [InstaPy/InstaPy#37]
Hi, I'm trying to use the example before starting to change anything. I change my user and pass and downloaded the chromedriver and put it in the assets folder. I am using Mac OSX 10.11.5.
I am getting this error
```
python quickstart.py 

Logged in successfully!

Traceback (most recent call last):
  File "quickstart.py", line 6, in <module>
    session.login()
  File "/Users/emiliomarin/Desktop/Fotos/Instagram/InstaPy-master/instapy/instapy.py", line 83, in login
    log_follower_num(self.browser, self.username)
  File "/Users/emiliomarin/Desktop/Fotos/Instagram/InstaPy-master/instapy/print_log_writer.py", line 10, in log_follower_num
    ('//a[@href="/' + username.lower()  + '/followers/"]')
  File "/Library/Python/2.7/site-packages/selenium-2.53.6-py2.7.egg/selenium/webdriver/remote/webdriver.py", line 293, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/Library/Python/2.7/site-packages/selenium-2.53.6-py2.7.egg/selenium/webdriver/remote/webdriver.py", line 752, in find_element
    'value': value})['value']
  File "/Library/Python/2.7/site-packages/selenium-2.53.6-py2.7.egg/selenium/webdriver/remote/webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "/Library/Python/2.7/site-packages/selenium-2.53.6-py2.7.egg/selenium/webdriver/remote/errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//a[@href="/havetstest/followers/"]"}
  (Session info: chrome=57.0.2987.133)
  (Driver info: chromedriver=2.29.461585 (0be2cd95f834e9ee7c46bcc7cf405b483f5ae83b),platform=Mac OS X 10.11.5 x86_64)
```

**Comment by emiliomarin** [comment]
I was trying it in a new account, tried it with one that already had followers and it worked

**Comment by timgrossmann** [comment]
@emiliomarin could you try it again with your new profile when replacing the content of print_log_writer.py with this, please?

```python
"""Module only used to log the number of followers to a file"""
from datetime import datetime
from selenium.common.exceptions import NoSuchElementException

def log_follower_num(browser, username):
  """Prints and logs the current number of followers to
  a seperate file"""
  browser.get('https://www.instagram.com/' + username)
  follower_num = 0

  try:
    follower_num = browser.find_element_by_xpath\
      ('//a[@href="/' + username.lower()  + '/followers/"]').text
  except NoSuchElementException:
    follower_num = 0

  with open('./logs/followerNum.txt', 'a') as numFile:
    numFile.write(datetime.now().strftime('%Y-%m-%d %H:%M '))
    numFile.write(str(follower_num) + '\n')
```

**Comment by timgrossmann** [comment]
If this works, #40 fixes the issue

**Comment by timgrossmann** [comment]
Fixed with #40
